### PR TITLE
Setting additional properties for 1.10.9 of ERT tile

### PR DESCRIPTION
**Pull request by bijukunjummen**
The changes are to account for new fields in 1.10.9 ERT tile. I have only accounted for parameters in AWS and GCP though, so likely an issue needs to be captured for vSphere and GCP

### CODE REVIEW

**Comment by bijukunjummen** [comment]
//cc @ryanpei

**Comment by ryanpei** [comment]
thanks @bijukunjummen I'll open up a story for vsphere and azure

**Comment by bijukunjummen** [comment]
Hold off merging this - seeing an issue  - https://concourse.customer0.io/teams/pcfs/pipelines/bk-cf-aws-install/jobs/deploy-ert/builds/7

**Comment by bijukunjummen** [comment]
This can be merged now @krishicks, @ryanpei - I have tested the flow for AWS. I have made the changes for GCP but I have not tested it yet as I don't have any environment to test this on - will reach out to @ryanpei

**Comment by ryanpei** [comment]
thanks @bijukunjummen i'll try it on my GCP environment today.

File: tasks/install-pcf-gcp/terraform/sql.tf [hunk 1]
```diff
@@ -96,6 +96,30 @@ resource "google_sql_database" "diego" {
   count = "1"
 }
 
+resource "google_sql_database" "account" {
+  name       = "diego"
```
Comment (krishicks) [inline tasks/install-pcf-gcp/terraform/sql.tf]: These names are all "diego".

File: tasks/install-pcf-gcp/terraform/sql.tf [hunk 2]
```diff
@@ -96,6 +96,30 @@ resource "google_sql_database" "diego" {
...
+  depends_on = ["google_sql_database.account"]
+  instance   = "${google_sql_database_instance.master.name}"
+
+  count = "1"
+}
+
+resource "google_sql_database" "networkpolicyserver" {
+  name       = "diego"
+  depends_on = ["google_sql_database.nfsvolume"]
```
Comment (krishicks) [inline tasks/install-pcf-gcp/terraform/sql.tf]: Is this really dependent on the other database?

Comment (bijukunjummen) [inline tasks/install-pcf-gcp/terraform/sql.tf]: No, but I see that it is being ordered by the previous database, I believe just an attempt to sequence it.

File: tasks/install-pcf-gcp/terraform/sql.tf (hunk 1 above)
Comment (bijukunjummen) [inline tasks/install-pcf-gcp/terraform/sql.tf]: Yes :-), copy-paste error, fixed it now

**Comment by abbyachau** [comment]
Thanks, @bijukunjummen. Closing this issue.

**Pull request merged**